com.microsoft.mrtk.input/Interactors/InteractorVisuals/MRTKLineVisual.cs

```diff
@@ -447,6 +447,33 @@ private void InitializeLineRendererProperties()
         /// </summary>
         private void LocateTargetHitPoint(SelectEnterEventArgs args)
         {
+            // If no hit interactable, abort
```

Comment:
There is also a LocateTargetHitPoint method like this in SpatialMouseInteractorCursorVisual. I can add these changes there too.

Reply:
is it worth trying to have them share? Maybe moving this method to BaseReticleVisual?

Reply:
I like this thinking.   Let's consider making this change after this PR and the spatial mouse change goes in:

- https://github.com/microsoft/MixedRealityToolkit-Unity/issues/11649